packages/components/src/ui/elevation/elevation.js

```diff
@@ -0,0 +1,44 @@
+/**
+ * External dependencies
+ */
+import { contextConnect } from '@wp-g2/context';
+
+/**
+ * Internal dependencies
+ */
+import { View } from '../view';
+import { useElevation } from './use-elevation';
+
+/**
+ *
+ * @param {import('@wp-g2/create-styles').ViewOwnProps<import('./types').Props, 'div'>} props
+ * @param {import('react').Ref<any>} forwardedRef
+ */
+function Elevation( props, forwardedRef ) {
+	const otherProps = useElevation( props );
+
+	return <View aria-hidden { ...otherProps } ref={ forwardedRef } />;
```

Comment:
Ooo! Thanks for the feedback. `Elevation` actually shouldn't accept `children` 😱 .

(@sarayourfriend how would we adjust the types for that? I think this may be one of the few new components that don't accept `children`)

Reply:
Oh, interesting. I'll do some tinkering.